docs/csslint-options.md

```diff
@@ -86,6 +86,35 @@ For the current csslint version, the following formatters are supported:
 
 For an explanation of those formatters, [check the csslint wiki](https://github.com/stubbornella/csslint/wiki/Command-line-interface), section Options/--format.
 
+#### customFormatter
```

Comment:
use `formatter` to match https://github.com/CSSLint/csslint/pull/593/files#diff-f8160de8a8ea5ad7eee349d59a5f227bR13